crypto/bls_multisig.go

```diff
@@ -238,12 +263,17 @@ func RemoveBLSPublicKeys(aggKey PublicKey, keysToRemove []PublicKey) (PublicKey,
 //
 // This is a special case function of VerifyBLSSignatureManyMessages, using a single
 // message and hasher.
+//
+// The function returns:
+//  - (false, invalidInputsError) if:
+//      - Or at least one key is not of type PubKeyBLSBLS12381
+//      - input hasher is nil or its output size is not 128 bytes
```

Comment:
This is a case where the interface i implemented by more than one structure. Therefore the conditions of returning certain errors aren't the same (some errors may even not be returned in some implementations). 
The 2 implementing structures are public, and I think that the caller should read their doc for details instead of the interface's doc. 
( I know that we usually duplicate interface and struct docs in our repo, but I believe in most of these cases, the interface is implemented by one struct and one mock)